src/main/java/io/getstream/chat/java/models/App.java

```diff
@@ -791,6 +814,107 @@ public AppGetRateLimitsRequest endpoint(@NotNull String endpoint) {
     }
   }
 
+  @Data
+  @NoArgsConstructor
+  public static class PushProvider {
```

Comment:
Why the list of fields is not complete ? Are we going to skip legacy fields like templates etc... ?

Reply:
Ignore me. I should be the same as we do in go-sdk.

Reply:
yup, same as Go